doc/data_structures.rst

```diff
@@ -133,7 +133,7 @@ know the dimensions or type of the tensor, and as a result, the
 tensor data.  The C++ function signature for ``get_tensor()`` is shown
 below.  Note that a pointer to the newly allocated data, tensor
 dimensions, and tensor type are returned to the user via
-modifying referenced variables that the user declares before the
+modifying reference variables that the user declares before the
```

Comment:
I think this one was correct as it was, that the variables were referenced, not that the variables were specifically references